pool/cqrs_command/smtp_command.ml

```diff
@@ -220,3 +220,24 @@ end = struct
 
   let effects = Email.Guard.Access.Smtp.update
 end
+
+module Delete : sig
+  include Common.CommandSig with type t = SmtpAuth.delete
+
+  val handle
+    :  ?tags:Logs.Tag.set
+    -> t
+    -> (Pool_event.t list, Pool_common.Message.error) result
+
+  val effects : SmtpAuth.Id.t -> Guard.ValidationSet.t
+end = struct
+  type t = SmtpAuth.delete
+
+  let handle ?(tags = Logs.Tag.empty) (command : t) =
+    let open CCResult in
+    Logs.info ~src (fun m -> m "Handle command Delete" ~tags);
+    Ok [ Email.SmtpDeleted command |> Pool_event.email ]
```

Comment:
One more event to add, as the "worker" is caching the smtp accounts:

`; clear_cache_event ()`